docs/index.bs

```diff
@@ -2891,7 +2891,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
 
       1. If |serviceWorker| is not running, abort these steps.
       1. Let |serviceWorkerGlobalScope| be |serviceWorker|'s [=service worker/global object=].
-      1. Set |serviceWorkerGlobalScope|'s closing flag to true.
+      1. Set |serviceWorkerGlobalScope|'s {{closing!!attribute}} flag to true.
```

Comment:
It's not an attribute (i.e. there is no `self.closing`); it's a flag. The reason this works is because of some faulty Bikeshed heuristics.

To make this work properly you need to edit the HTML spec to add `data-export="" data-dfn-for="WorkerGlobalScope"` to the `<dfn>`. Then you can do `[=WorkerGlobalScope/closing=]` in this document.

Alternately you can use `<pre class="anchors">` to manually add something that isn't exported, but the best fix is to export it.